test/e2e/persistent_volumes.go

```diff
@@ -662,3 +812,94 @@ func makeWritePod(ns string, pvcName string) *api.Pod {
 		},
 	}
 }
+
+// Generate a list of PV specs.
```

Comment:
Add comment here and in makePersistentVolume() that the reclaimPolicy is set to "Recycle" so that the nfs recycler is tested when the pvc is deleted.